CHANGELOG

```diff
@@ -1,6 +1,8 @@
 devel
 -----
 
+* Activate RDB_CoveringIterator and use it for some geo index queries.
```

Comment:
Because the CHANGELOG is end-user-visible, what is the benefit of the RDB_CoveringIterator for end users?
Is there any way to see from query execution plans that an RDB_CoveringIterator will be used?
CHANGELOG entry does not necessarily need to be adjusted, but I am just curious what the RDB_CoveringIterator can do for me. :sweat_smile:

Reply:
At least you don't need to sort results by distance to their centroids